my_game.py

```diff
@@ -392,89 +419,122 @@ def setup(self):
         # setup spawn_ufo to run regularly
         arcade.schedule(self.spawn_ufo, UFO_SPAWN_RATE)
 
+        # create UI objs
+        self.title_graphics = arcade.load_texture("images/UI/asteroidsTitle.png")
+        self.play_button = arcade.load_texture("images/UI/asteroidsStartButton.png")
+
     def on_draw(self):
         """
         Render the screen.
         """
 
+        # always
+
         # This command has to happen before we start drawing
         arcade.start_render()
-        
-        # Draw asteroids
-        self.asteroid_list.draw()
 
-        # Draw the player shot
-        self.player_shot_list.draw()
+        # only in intro
+        if self.game_state == GameState.INTRO:
 
-        # Draw the player sprite
-        self.player_sprite.draw()
+            self.title_graphics.draw_scaled(
+                center_x=TITLE_X,
+                center_y=TITLE_Y
+            )
 
-        # draw ufo(s)
-        self.ufo_list.draw()
+            self.play_button.draw_scaled(
+                center_x=PLAY_BUTTON_X,
+                center_y=PLAY_BUTTON_Y,
+            )
 
-        # and their shots
-        self.ufo_shot_list.draw()
+        # only in-game
+        elif self.game_state == GameState.IN_GAME:
 
-        # Draw players score on screen
-        arcade.draw_text(
-            "SCORE: {}".format(self.player_score),  # Text to show
-            10,  # X position
-            SCREEN_HEIGHT - 20,  # Y_position
-            arcade.color.WHITE  # Color of text
-        )
-        arcade.draw_text(
-            "LIVES: {}".format(self.player_sprite.lives),  # Text to show
-            10,  # X position
-            SCREEN_HEIGHT - 45, # Y positon
-            arcade.color.WHITE  # Color of text
-        )
+            # Draw the player shot
+            self.player_shot_list.draw()
+
+            # Draw the player sprite
+            self.player_sprite.draw()
+
+            # Draw asteroids
+            self.asteroid_list.draw()
+
+            # draw ufo(s)
+            self.ufo_list.draw()
+
+            # and their shots
+            self.ufo_shot_list.draw()
+
+            # Draw players score on screen
+            arcade.draw_text(
+                "SCORE: {}".format(self.player_score),  # Text to show
+                10,  # X position
+                SCREEN_HEIGHT - 20,  # Y_position
+                arcade.color.WHITE  # Color of text
+            )
+            arcade.draw_text(
+                "LIVES: {}".format(self.player_sprite.lives),  # Text to show
+                10,  # X position
+                SCREEN_HEIGHT - 45, # Y positon
+                arcade.color.WHITE  # Color of text
+            )
 
```

Comment:
Maybe you could put in a placeholder here for state _GAME_OVER_? Just put _pass_ in the _elif_ block.